packages/types/src/types/extrinsic.ts

```diff
@@ -159,6 +160,7 @@ export interface SignatureOptions {
   signedExtensions?: string[];
   signer?: Signer;
   tip?: AnyNumber;
+  assetId: Option<AssetId>;
```

Comment:
```suggestion
  assetId?: AnyNumber;
```

In the JSON itself it appears as a number.